LeadKit/LeadKit/Extensions/Observable/Observable+ToastErrorLogging.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Copyright (c) 2017 Touch Instinct
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import RxSwift
+import Toast_Swift
+
+public extension Observable {
+
+    /// Method which shows toast with localized description of error in DEBUG mode
+    ///
+    /// - Returns: The source sequence with the side-effecting behavior applied.
+    func showErrorsInToastInDebugMode() -> Observable<Observable.E> {
+        return `do`(onError: { (error) in
+            #if DEBUG
+                UIApplication.shared.keyWindow?.makeToast(error.localizedDescription)
+            #endif
+        })
+    }
```

Comment:
if no DEBUG - will be compile or analiazator err on an unused parameter 'error' ?
and where we are going to use it? Maybe alerts for testers also needed?